gradle-plugin/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id 'java-gradle-plugin'
+}
+
+java {
+  sourceCompatibility = JavaVersion.VERSION_1_8
+  targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+task copyJar(type: Copy) {
+  from "build/libs/gradle-plugin.jar"
+  into "../gradle-server/src/main/resources"
```

Comment:
what's this task for? and why target folder is `src/main/resources`? just curious about how they work together, can you briefly describe it?

Reply:
this task is to copy the generated plugin jar to `resources` folder in `gradle-server` project. In [pr887](https://github.com/microsoft/vscode-gradle/pull/887/files#diff-95b98e0f2246374e12d3f8a536ab328cf80e6edee8a9609d3becea4568cd8bd0R96), we can use `getResourceAsStream()` to directly get the plugin jar file, copy it to the temp path and add it to the classpath of the gradle daemon to apply this plugin.